registration/frontend/main.css

```diff
@@ -0,0 +1,17 @@
+<style type="text/css">
```

Comment:
You don't need these style tags! Please reference this file as an example:
https://github.com/SpectraHackathon/spectra-revamp/blob/master/team/frontend/main.css